common/app/model/EmailNewsletters.scala

```diff
@@ -2,419 +2,417 @@ package model
 
 import controllers.ListIds
 
-case class EmailSubscriptions(subscriptions: List[EmailSubscription])
-case class EmailSubscription(
+case class EmailNewsletters(subscriptions: List[EmailNewsletter])
```

Comment:
Do we need this case class? Could we just use `List[EmailNewsletter]` instead and avoid a level of indirection? Lines like `availableLists.subscriptions` below seem much less clear than just `availableLists`